.travis.yml

```diff
@@ -201,8 +201,6 @@ before_install:
 - $PY_EXE -m pip --version
 # setuptools may be out of date on osx
 - $PY_EXE -m pip install --user -U pip setuptools wheel
-# ensure python bin dir exists (and coverage dependencies installed)
-- $PY_EXE -m pip install --user -U nose codecov coveralls requests
```

Comment:
Why is this done pre-installation? Probably that was important but I ignored it...

Reply:
can't see why it'd matter, but probably doesn't do any harm either